src/bucklescript/output_bucklescript_module.re

```diff
@@ -291,7 +358,24 @@ let generate_operation = config =>
       structure,
     );
 
-let generate_modules = (config, operations) => {
-  let generated = List.map(generate_operation(config), operations);
-  Mod.mk(Pmod_structure(List.concat(generated)));
+let generate_modules = (config, module_definition, operations) => {
+  switch (operations) {
+  | [] => []
+  | [a] =>
```

Comment:
Can you reproduce this. I think by default a single fragment creates a module for that fragment. Only if you explicitly inline with the `inline` option, it will put the fragment in the current module (to prevent two layers of modules in some cases). I have a lot of fragments in the app and have not run into issues with this.

Reply:
```reason
module Fragments = [%graphql
  {|
  fragment ListFragment on Lists {
    nullableOfNullable
    nullableOfNonNullable
  }
|}
];
```

IMO this ListFragment should be available via: `Fragments.ListFragment` and not just `Fragments`. This was previous behaviour in the `test_bucklescript/__tests__/fragmentDefinition.re`.